kolibri/core/kolibri_plugin.py

```diff
@@ -15,17 +12,11 @@ class KolibriCore(KolibriPluginBase):
     pass
 
 
-class MainNavigationItem(hooks.NavigationHook):
-    label = _("Do nothing")
-    url = '#'
-
-
```

Comment:
I added a "Technical debt (clean up post-MVP)" list on the tasks board for tracking things we'll want to revisit later, and added a card for the pluggable nav.